packages/redis/src/config.ts

```diff
@@ -58,11 +58,59 @@ export class RedisOptions implements IRedisOptions {
         return config.noop();
     }
 
-    @config.field(config.converters.listOf(config.converters.string))
-    public set readStream(_: string[]) {
+    @config.field(config.converters.string)
+    public set readStream(_: string) {
+        config.noop();
+    }
+    public get readStream(): string {
+        return config.noop();
+    }
+
+    @config.field(config.converters.string)
+    public set consumerGroup(_: string) {
+        config.noop();
+    }
+    public get consumerGroup(): string {
+        return config.noop();
+    }
+
+    @config.field(config.converters.string)
+    public set consumerId(_: string) {
+        config.noop();
+    }
+    public get consumerId(): string {
+        return config.noop();
+    }
+
+    @config.field(config.converters.string)
+    public set consumerGroupStartId(_: string) {
+        config.noop();
+    }
+    public get consumerGroupStartId(): string {
+        return config.noop();
+    }
+
+    @config.field(config.converters.number)
```

Comment:
can you use `config.converts.timeSpan` for all timings and remove the `Ms` suffix? the converter allows ppl to put something like `idleTimeout: "1h"` into the config.